Tests/js/views/services/assets/ez-dashboardblocksviewservice-tests.js

```diff
@@ -0,0 +1,186 @@
+
+    successTest = new Y.Test.Case({
+        name: 'eZ Dashboard Blocks View Service load success test',
+
+        setUp: function () {
+            this.app = new Y.Mock();
+            this.capi = new Y.Mock();
+            this.discoveryService = new Y.Mock();
+            this.rootLocationModel = new Y.Mock();
+
+            Y.Mock.expect(this.capi, {
+                method: 'getDiscoveryService',
+                run: Y.bind(function () {
+                    return this.discoveryService;
+                }, this)
+            });
+
+            this.service = new Y.eZ.DashboardBlocksViewService({
+                app: this.app,
+                capi: this.capi,
+                rootLocation: this.rootLocationModel
+            });
+        },
+
+        tearDown: function () {
+            this.service.destroy();
+        },
+
+        'Should load root location data when being loaded': function () {
+            var rootLocationId = 'root-location',
+                loadLocationResponse = {
+                    document: {
+                        Location: {}
+                    }
+                };
+
+            Y.Mock.expect(this.discoveryService, {
+                method: 'getInfoObject',
+                args: ['rootLocation', Y.Mock.Value.Function],
+                run: function (identifier, callback) {
+                    callback(false, {
+                        _href: rootLocationId
+                    });
+                }
+            });
+
+            Y.Mock.expect(this.rootLocationModel, {
+                method: 'set',
+                args: ['id', rootLocationId],
+                returns: this.rootLocationModel
+            });
+
+            Y.Mock.expect(this.rootLocationModel, {
+                method: 'load',
+                args: [Y.Mock.Value.Object, Y.Mock.Value.Function],
+                run: Y.bind(function (options, callback) {
+                    Y.Assert.areSame(this.capi, options.api, 'Should pass correct API to the `load` method of root location model');
+
+                    callback(false, loadLocationResponse);
+                }, this)
+            });
+
+            this.service.load(Y.bind(function () {
+                this.resume(Y.bind(function () {
+                    Y.Mock.verify(this.rootLocationModel);
+                    Y.Mock.verify(this.capi);
+                    Y.Mock.verify(this.discoveryService);
+                }, this));
+            }, this));
+            this.wait();
+        }
+    });
+
+    errorTests = new Y.Test.Case({
+        name: 'eZ Dashboard Blocks View Service load error handling tests',
+
+        setUp: function () {
+            this.app = new Y.Mock();
+            this.capi = new Y.Mock();
+            this.discoveryService = new Y.Mock();
+            this.rootLocationModel = new Y.Mock();
+
+            Y.Mock.expect(this.capi, {
+                method: 'getDiscoveryService',
+                run: Y.bind(function () {
```

Comment:
same nitpick as in https://github.com/ezsystems/PlatformUIBundle/pull/589/files#r65530839